docs/build/guides/transactions/simulateTransaction-Deep-Dive.mdx

```diff
@@ -375,11 +375,11 @@ Soroban smart contracts on Stellar use a multidimensional resource fee model, ch
 
 Fees are calculated based on the resource consumption declared in the transaction. Refundable fees are charged before execution and refunded based on actual usage, while non-refundable fees are calculated from CPU instructions, read bytes, write bytes, and transaction size. [Check out this document for a more in depth understanding of fees and metering.](https://developers.stellar.org/docs/learn/fundamentals/fees-resource-limits-metering)
 
-## Preflight Error Handling
+## Error Handling
 
-The preflight mechanism provides an estimation of CPU and memory consumption in a transaction. It helps identify potential errors and resource limitations before actual submission. Errors returned by the host are propagated through RPC and do not cover network errors or errors in RPC itself.
+The simulate transaction mechanism provides an estimation of CPU and memory consumption in a transaction. It helps identify potential errors and resource limitations before actual submission. Errors returned by the host are propagated through RPC and do not cover network errors or errors in RPC itself.
```

Comment:
“Transaction simulation mechanism” rolls better, not sure it’s a 1:1 replacement for “preflight” in that way.